### DISCUSSION TITLE
Dot file for ignoring folder

### DISCUSSION THREAD

**Discussion by momvart** [microsoft/pyright#7810] [Q&A]
Hi.
I'm wondering if there's a certain dot file that you use to ignore a directory. I know it's possible to exclude paths using the config files but I want something simpler, just a marker.

Particularly in my case, I have a large project which has a `scripts` folder containing python files. I don't want the root folder to be scanned but only the `scripts`. I imagine with such a dot file I can create one in the root folder, and add a `pyproject.toml` in `scripts` subfolder.

**Comment by erictraut** [comment]
If you want to include only a specific subdirectory within your project, you can use the "include" configuration setting to specify that directory.

```json
{
    "include": ["scripts"]
}
```

**Reply by momvart** [reply]
Thanks. Yeah it works.

However, it's probably not the best if you don't wanna pollute the root folder of your project with a python configuration file just pointing to a subfolder.
